icons/ribbon.svg

```diff
@@ -9,9 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M17.75 9.01c-.52 2.08-1.83 3.64-3.18 5.49l-2.6 3.54-2.97 4-3.5-2.54 3.85-4.97c-1.86-2.61-2.8-3.77-3.16-5.44" />
-  <path d="M17.75 9.01A7 7 0 0 0 6.2 9.1C6.06 8.5 6 7.82 6 7c0-3.5 2.83-5 5.98-5C15.24 2 18 3.5 18 7c0 .73-.09 1.4-.25 2.01Z" />
+  <path d="M11.973 18.036 14.57 14.5c1.35-1.85 2.66-3.41 3.18-5.49A7 7 0 0 0 6.2 9.1C6.06 8.5 6 7.82 6 7c0-3.5 2.83-5 5.98-5C15.24 2 18 3.5 18 7a8 8 0 0 1-.25 2.01" />
+  <path d="M11.99 11.22C11 10 10 9 10 8a2 2 0 1 1 4 0c0 1-1 2-2.01 3.22" />
+  <path d="m14.57 14.5 3.284 4.179a1 1 0 0 1-.203 1.43l-1.894 1.36a1 1 0 0 1-1.384-.215L11.97 18.04l-2.38 3.206a1 1 0 0 1-1.39.213l-1.865-1.353a1 1 0 0 1-.203-1.422L9.35 14.53c-1.86-2.61-2.8-3.77-3.16-5.44" />
   <path d="m9.35 14.53 2.64-3.31" />
```

Comment:
Let's elongate the bottom strips for better vertical centering:
```suggestion
  <path d="M12 11.22C11 9.997 10 9 10 8a2 2 0 0 1 4 0c0 1-.998 2.002-2.01 3.22" />
  <path d="m12 18 2.57-3.5" />
  <path d="M6.243 9.016a7 7 0 0 1 11.507-.009" />
  <path d="M9.35 14.53 12 11.22" />
  <path d="M9.35 14.53C7.728 12.246 6 10.221 6 7a6 5 0 0 1 12 0c-.005 3.22-1.778 5.235-3.43 7.5l3.557 4.527a1 1 0 0 1-.203 1.43l-1.894 1.36a1 1 0 0 1-1.384-.215L12 18l-2.679 3.593a1 1 0 0 1-1.39.213l-1.865-1.353a1 1 0 0 1-.203-1.422z" />
```
<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M12+11.22C11+9.997+10+9+10+8a2+2+0+0+1+4+0c0+1-.998+2.002-2.01+3.22%22+/%3E%3Cpath+d=%22m12+18+2.57-3.5%22+/%3E%3Cpath+d=%22M6.243+9.016a7+7+0+0+1+11.507-.009%22+/%3E%3Cpath+d=%22M9.35+14.53+12+11.22%22+/%3E%3Cpath+d=%22M9.35+14.53C7.728+12.246+6+10.221+6+7a6+5+0+0+1+12+0c-.005+3.22-1.778+5.235-3.43+7.5l3.557+4.527a1+1+0+0+1-.203+1.43l-1.894+1.36a1+1+0+0+1-1.384-.215L12+18l-2.679+3.593a1+1+0+0+1-1.39.213l-1.865-1.353a1+1+0+0+1-.203-1.422z%22+/%3E&name=image"><img alt="icons" width="400" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIgMTEuMjJDMTEgOS45OTcgMTAgOSAxMCA4YTIgMiAwIDAgMSA0IDBjMCAxLS45OTggMi4wMDItMi4wMSAzLjIyIiAvPgogIDxwYXRoIGQ9Im0xMiAxOCAyLjU3LTMuNSIgLz4KICA8cGF0aCBkPSJNNi4yNDMgOS4wMTZhNyA3IDAgMCAxIDExLjUwNy0uMDA5IiAvPgogIDxwYXRoIGQ9Ik05LjM1IDE0LjUzIDEyIDExLjIyIiAvPgogIDxwYXRoIGQ9Ik05LjM1IDE0LjUzQzcuNzI4IDEyLjI0NiA2IDEwLjIyMSA2IDdhNiA1IDAgMCAxIDEyIDBjLS4wMDUgMy4yMi0xLjc3OCA1LjIzNS0zLjQzIDcuNWwzLjU1NyA0LjUyN2ExIDEgMCAwIDEtLjIwMyAxLjQzbC0xLjg5NCAxLjM2YTEgMSAwIDAgMS0xLjM4NC0uMjE1TDEyIDE4bC0yLjY3OSAzLjU5M2ExIDEgMCAwIDEtMS4zOS4yMTNsLTEuODY1LTEuMzUzYTEgMSAwIDAgMS0uMjAzLTEuNDIyeiIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>

(I've also done some manual optimization or some elliptic arcs and overall better curvature)